docs/script-api.md

```diff
@@ -1,6125 +1,2 @@
-- [`float target_x`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=target_x) &SpecialShot::target_x
-- [`float target_y`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=target_y) &SpecialShot::target_y
-### `SoundShot`
-Derived from [`Entity`](#entity) [`Movable`](#movable) [`LightShot`](#lightshot)
-### `Spark`
-Derived from [`Entity`](#entity) [`Movable`](#movable) [`Flame`](#flame)
-- [`ParticleEmitterInfo particle`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=particle) &Spark::particle
-- [`Entity fx_entity`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=fx_entity) &Spark::fx_entity
-- [`float rotation_center_x`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=rotation_center_x) &Spark::rotation_center_x
-- [`float rotation_center_y`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=rotation_center_y) &Spark::rotation_center_y
-- [`float rotation_angle`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=rotation_angle) &Spark::rotation_angle
-- [`float size`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=size) &Spark::size
-\
-slowly goes down to default 1.0, is 0.0 when not on screen
-- [`float size_multiply`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=size_multiply) &Spark::size_multiply
-\
-0.0 when not on screen
-- [`float next_size`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=next_size) &Spark::next_size
-\
-width and height will be set to this value  size_multiply next frame
-- [`int size_change_timer`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=size_change_timer) &Spark::size_change_timer
-\
-very short timer before next size change, giving a pulsing effect
-### `TiamatShot`
-Derived from [`Entity`](#entity) [`Movable`](#movable) [`LightEmitter`](#lightemitter)
-### `Fireball`
-Derived from [`Entity`](#entity) [`Movable`](#movable) [`LightShot`](#lightshot) [`SoundShot`](#soundshot)
-- [`ParticleEmitterInfo particle`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=particle) &Fireball::particle
-### `Leaf`
-Derived from [`Entity`](#entity) [`Movable`](#movable)
-- [`float fade_away_counter`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=fade_away_counter) &Leaf::fade_away_counter
-\
-counts to 100.0 then the leaf fades away
-- [`int swing_direction`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=swing_direction) &Leaf::swing_direction
-- [`bool fade_away_trigger`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=fade_away_trigger) &Leaf::fade_away_trigger
-### `AcidBubble`
-Derived from [`Entity`](#entity) [`Movable`](#movable)
-- [`float speed_x`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=speed_x) &AcidBubble::speed_x
-- [`float speed_y`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=speed_y) &AcidBubble::speed_y
-- [`float float_counter`](https://github.com/spelunky-fyi/overl
```

Comment:
I thought the idea was to keep this and add wiki?

Reply:
There's no need to keep this though. Any good reason?

Reply:
Wait... Really?

Reply:
Well that sucks, because the Wiki is better at everything else... If we structure it well you won't have any difficulties to find the right path though on your own.

Reply:
The hamburger menu was the only way to find anything in script-api, taking it away completely is not a good option before a new actual doc platform like Slate is up with proper search across all pages.

Reply:
Most of my interaction with script-api is checking the function parameters with simple Ctrl+F. 
Even with best tree structure the wiki will be a annoying. But that could be just 'me' problem

Reply:
Wiki will be a better alternative for browsing. Marginally, global functions is still 47 pages printed compared to 207 pages of script-api.